File: Status_21_07_2022/pattern7.py
#!/usr/bin/python3

def pattern(row):
    i = 1
    while (i <= row):
        j = i
        while(j > 0):
            print(row-j+1,end = " ")
            j -=1
        i +=1
        print()

pattern(int(input("Enter the number of rows: ")))

File: Status_21_07_2022/pattern7.c
#include <stdio.h>

void pattern(int row)
{
    int i = 0;
    while(i<row)
    {
        int j = i;
        while(j >=0)
        {
            printf("%d ",row - j--);
        }
        printf("\n");
        i++;
    }
}

int main()
{
    int num = 0;
    printf("Enter the number of rows: ");
    scanf("%d",&num);
    pattern(num);
}